apps/hpc-cdm/src/app/components/enketo/formToolbar/indicator.tsx

```diff
@@ -120,8 +120,7 @@ const Indicator = (props: Props) => {
               .replace('{person}', lastUpdatedBy)}
           </p>
         </StatusDetails>
-      }
-    >
+      }>
```

Comment:
If you want `>` to go on next line, like it used to, you could use Prettier config `bracketSameLine: false`.